Node 16 support in macOS

I have an error when nodejs 16 in macOS.

```

Failed to execute '/opt/homebrew/Cellar/node/16.1.0/bin/node /Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release/opencc.node --module_name=opencc --module_path=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@0.14.0
node-pre-gyp info using node@16.1.0 | darwin | arm64
[opencc] Removing "/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release"
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@16.1.0 | darwin | arm64
gyp info ok
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@16.1.0 | darwin | arm64
gyp info find Python using Python version 3.8.2 found at "/Applications/Xcode.app/Contents/Developer/usr/bin/python3"
(node:81779) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
(Use `node --trace-deprecation ...` to show where the warning was created)
gyp info spawn /Applications/Xcode.app/Contents/Developer/usr/bin/python3
gyp info spawn args [
gyp info spawn args   '/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ut/Library/Caches/node-gyp/16.1.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/ut/Library/Caches/node-gyp/16.1.0',
gyp info spawn args   '-Dnode_gyp_dir=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/ut/Library/Caches/node-gyp/16.1.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info ok
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@16.1.0 | darwin | arm64
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  COPY Release/hk2s.json
  COPY Release/hk2t.json
  COPY Release/jp2t.json
  COPY Release/s2hk.json
  COPY Release/s2t.json
  COPY Release/s2tw.json
  COPY Release/s2twp.json
  COPY Release/t2hk.json
  COPY Release/t2jp.json
  COPY Release/t2s.json
  COPY Release/t2tw.json
  COPY Release/tw2s.json
  COPY Release/tw2sp.json
  COPY Release/tw2t.json
  TOUCH Release/obj.target/configs.stamp
  CXX(target) Release/obj.target/opencc/node/marisa.o
  CXX(target) Release/obj.target/opencc/node/opencc.o
In file included from ../node/opencc.cc:2:
In file included from ../../nan/nan.h:56:
In file included from /Users/ut/Library/Caches/node-gyp/16.1.0/include/node/node.h:63:
In file included from /Users/ut/Library/Caches/node-gyp/16.1.0/include/node/v8.h:30:
/Users/ut/Library/Caches/node-gyp/16.1.0/include/node/v8-internal.h:452:38: error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
            !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
                                ~~~~~^~~~~~~~~~~
                                     remove_cv
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/type_traits:697:50: note: 'remove_cv' declared here
template <class _Tp> struct _LIBCPP_TEMPLATE_VIS remove_cv
                                                 ^
1 error generated.
make: *** [Release/obj.target/opencc/node/opencc.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (node:events:365:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
gyp ERR! System Darwin 20.3.0
gyp ERR! command "/opt/homebrew/Cellar/node/16.1.0/bin/node" "/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-gyp/bin/node-gyp.js" "build" "--module=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release/opencc.node" "--module_name=opencc" "--module_path=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release" "--napi_version=8" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v93"
gyp ERR! cwd /Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc
gyp ERR! node -v v16.1.0
gyp ERR! node-gyp -v v7.1.2
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/opt/homebrew/Cellar/node/16.1.0/bin/node /Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-gyp/bin/node-gyp.js build --module=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release/opencc.node --module_name=opencc --module_path=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:365:28)
node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1067:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
node-pre-gyp ERR! System Darwin 20.3.0
node-pre-gyp ERR! command "/opt/homebrew/Cellar/node/16.1.0/bin/node" "/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-pre-gyp/bin/node-pre-gyp" "rebuild"
node-pre-gyp ERR! cwd /Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc
node-pre-gyp ERR! node -v v16.1.0
node-pre-gyp ERR! node-pre-gyp -v v0.14.0
node-pre-gyp ERR! not ok
Failed to execute '/opt/homebrew/Cellar/node/16.1.0/bin/node /Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/node-gyp/bin/node-gyp.js build --module=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release/opencc.node --module_name=opencc --module_path=/Users/ut/actions-runner/_work/build-apps/build-apps/project/node_modules/opencc/build/Release --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
```

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -9,6 +9,7 @@ environment:
   matrix:
     # Build Node.js
     - nodejs_version: stable
+    - nodejs_version: 16
     - nodejs_version: 14
     - nodejs_version: 12
     - nodejs_version: 10

--- a/.github/workflows/nodejs.yml
+++ b/.github/workflows/nodejs.yml
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [10.x, 12.x, 14.x, 16.x]
 
     steps:
     - uses: actions/checkout@v2

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,6 +6,7 @@ cache:
 
 node_js:
   - stable
+  - 16
   - 14
   - 12
   - 10

--- a/node/global.gypi
+++ b/node/global.gypi
@@ -18,7 +18,7 @@
         'xcode_settings': {
           'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
           'MACOSX_DEPLOYMENT_TARGET': '10.7',
-          'OTHER_CPLUSPLUSFLAGS': ["-std=c++11", "-stdlib=libc++"],
+          'OTHER_CPLUSPLUSFLAGS': ["-std=c++14", "-stdlib=libc++"],
           'OTHER_LDFLAGS': ["-stdlib=libc++"]
         }
       }],

